website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -68,6 +68,68 @@ edge can only be installed in Windows with administrator permissions
 
 Therefore, administrator permissions are required to install Edge in Windows automatically through Selenium Manager, and Edge is eventually installed in the usual program files folder (e.g., `C:\Program Files (x86)\Microsoft\Edge`).
 
+## Example WebDriver Code
+
+These are provided in C# but similar code would work for other languages. 
+
+If you want to use ChromeDriver for the version currently installed on your machine, then you can run:
```

Comment:
But this is just a normal code execution. Feels weird to put code that shows starting a driver here because there is nothing explicitly Selenium Manager related.

Reply:
My point was to drive home the point that the manager is operating in the background and make it clearer that normal driver execution is using the Selenium Manager. While this article says that, I felt it was challenging to understand as someone who has been having to manage driver metadata in a professional capacity.

Reply:
If it were up to me this whole page would be much shorter. We don't spend much time walking through implementation of any other feature. It's all supposed to just work in the background.

I'm not sure how to help people more if they can't read the first thing on the page: 😦 

> Selenium Manager is a command-line tool implemented in Rust that provides automated driver and browser management for Selenium. Selenium bindings use this tool by default, so you do not need to download it or add anything to your code or do anything else to use it.

Like, if we need to put that disclaimer in more places on the page, I'd be more in favor of that than just writing code that is the equivalent of "ignore this, just use Selenium like normal and it works"